Added Dark Mode Feature

KyleKent1223 opened the pull request:
Add dark mode toggle using JavaScript

KyleKent1223 committed:
Added Dark Mode Feature

Add dark mode toggle using JavaScript

MisuClick commented:
edit the toggle button

KyleKent1223 committed:
Edited Dark Mode

making the dark mode button simpler.

MisuClick commented:
edit the dark mode so that if you changed a page it will save the current status of the toggled mode(if dark mode and clicked the contact the mode should stay to dark).

KyleKent1223 committed:
Edit the Dark Mode Status

Resolved the issue, where the dark mod will return to default when changing the pages.

MisuClick commented:
gooods no probs
